src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import Layout from 'components/Layout';
+import 'styles/App.css';
```

Comment:
- [ ] due to there not being an App.css in the styles folder the user receives below error when trying to open your application.
![afbeelding](https://user-images.githubusercontent.com/39240820/230649518-46804169-9da0-4055-a0c9-fbc14fedbe38.png)

Reply:
@mikethreels Thanks for pointing out, I will make it right